integration-ids.conf

```diff
@@ -17,6 +17,7 @@ export SLACK_API_ALARMS_DATA_QUERY=106657
 export SLACK_API_ALARMS_FACILITIES=107438
 export SLACK_API_ALARMS_FACILITIES_COLLECTOR=108531
 export SLACK_API_ALARMS_INFRASTRUCTURE=107273
+export SLACK_API_ALARMS_PGD_SANDBOX=110372
```

Comment:
`SLACK_API_ALARMS_PGD` is sufficient here. The channel will also receive production alerts when we add them later.